source/features/update-pr-from-base-branch.tsx

```diff
@@ -21,14 +26,8 @@ async function mergeBranches(): Promise<AnyObject> {
 	});
 }
 
-async function handler(): Promise<void> {
-	const {base, head} = getBranches();
-	if (!confirm(`Merge the ${base.local} branch into ${head.local}?`)) {
-		return;
-	}
-
-	features.unload(import.meta.url);
-
+async function handler(event: DelegateEvent<MouseEvent, HTMLButtonElement>): Promise<void> {
+	event.delegateTarget.disabled = true;
```

Comment:
Not yet tested

Reply:
It works but we actually need to remove it after, or else:

![fixed](https://user-images.githubusercontent.com/1402241/234484770-de9607c7-bc71-413e-8396-3d08c93124d5.gif)

I added a `.remove()`